### PR TITLE
Reset text fields on close modals

### PR DESCRIPTION
**Pull request by wisniewski-patryk**

### CODE REVIEW

File: ShiftPlan.Blazor.Client/Pages/Components/Empoyees/AddEmployeeModal.razor [hunk 1]
```diff
@@ -9,7 +9,7 @@
 	<Icon Name="IconName.Plus"/> Add new employee
 </Button>
 
-<Modal @ref="modal" Title="Add employee" ShowCloseButton="true" IsVerticallyCentered="true">
+<Modal @ref="modal" Title="Add employee" ShowCloseButton="true" IsVerticallyCentered="true" OnHidden="ResetEmployeeName">
```
Comment (wisniewski-patryk) [inline ShiftPlan.Blazor.Client/Pages/Components/Empoyees/AddEmployeeModal.razor:12]: test

File: ShiftPlan.Blazor.Client/Pages/Components/UserIdentity/UserComponent.razor [hunk 2]
```diff
@@ -41,16 +41,16 @@
 	</FooterTemplate>
 </Modal>
 
-<Modal @ref="registerModal" Title="Register" ShowCloseButton="true">
+<Modal @ref="registerModal" Title="Register" ShowCloseButton="true" OnHidden="ResetInputFields">
 	<BodyTemplate>
 		<p>Password should contain at least 8 characters, including at least one uppercase letter, one number, and one non alphanumeric character.</p>
```
Comment (wisniewski-patryk) [inline ShiftPlan.Blazor.Client/Pages/Components/UserIdentity/UserComponent.razor:46]: test 2

**Review by MariuszMalec — COMMENTED** [review]
image
If I try to login and add "1" as suffix to email text I login succesful :-)

**Review by MariuszMalec — COMMENTED** [review]
Also register and if I add as suffix "1
" to email which was registered yet I will get this message 
image

**Comment by wisniewski-patryk** [comment]
On my machine works fine? :o

**Comment by MariuszMalec** [comment]
hmm, on my machine works not properly.

**Comment by wisniewski-patryk** [comment]
yeah, on me laptop everything works properly

I register mail3@mail.com, i can login, but trying mail3@mail.com1 I get 401 - Unauthorized

i think we can merge it

**Review by MariuszMalec — APPROVED** [review]

**Pull request merged**